app-frontend/src/app/pages/login/login.html

```diff
@@ -0,0 +1,29 @@
+<div>
+  <!-- Images (and assets) are parsed and loaded from within the public directory -->
+  <div class="block login-block">
+		<img ng-attr-src="{{::$ctrl.assetLogo}}">
```

Comment:
There's actually a whole lot of tabs in all the Sass files too. I wonder if we should quickly run a sed script on the whole directory to convert all tabs to spaces.